lib/compiler/src/v3_core.erl

```diff
@@ -944,6 +945,9 @@ verify_suitable_fields([{bin_element,_,Val,SzTerm,Opts}|Es]) ->
     end,
     {unit,Unit} = keyfind(unit, 1, Opts),
     case {SzTerm,Val} of
+	{{atom,_,undefined},{string,_,_}} ->
+	    %% UTF-8/16/32.
+	    ok;
 	{{atom,_,undefined},{char,_,_}} ->
```

Comment:
This clause is no longer covered by the test suite. Please make sure it is covered. (Hint: a good place could be `bs_utf_SUITE:literals/1`; construct a binary such as <<$a/utf8>>.)